src/test/java/io/strimzi/systemtest/DrainCleanerST.java

```diff
@@ -47,19 +57,26 @@ void testEvictionRequestOnKafkaPod() {
     @Test
     void testEvictionRequestOnRandomPod() {
```

Comment:
Why we need to change this test?

Reply:
because it need to match with the new code. 
as it was failing

Reply:
Here you can simply create an overload version of `createStatefulSetAndPDBWithWait` which takes just the `stsName` as input parameter and call the new version passing `stsName` an empty map of labels. That way, you can reuse the old `testEvictionRequestOnRandomPod` code.

Additionally, using plural is more appropriate for a map, so I would use `labels` instead of `label` everywhere.